lib/nice_partials/monkey_patch.rb

```diff
@@ -48,8 +48,8 @@ def p(*args)
 
   # Overrides `ActionView::Helpers::RenderingHelper#render` to push a new `nice_partial`
   # on the stack, so rendering has a fresh `partial` to store content in.
-  def render(*)
-    __partials.prepend nice_partial
+  def render(*, local_assigns)
```

Comment:
Does this need a double splat?

Reply:
I updated the signature follow Action View more closely since it has two render call styles. E.g. `render "partial", some: :locals` and `render partial: "partial", locals: { some: :locals }`.